<% if @subtask.errors.any? %>
<div class="alert alert-secondary">
<%= @subtask.errors.count %>件エラーが発生しました。以下ご確認お願いいたします。
  <ul>
    <% @subtask.errors.full_messages.each do |message| %>
    <li><%= message %></li>
    <% end %>
  </ul>
</div>
<% end %>
<%= form_with model: @subtask, url: task_subtasks_path(@task.id), local: true do |f| %>
  <div class="form-group">
    <%= f.label :サブタスク %>
    <%= f.text_area :subtask_content, autofocus: true, class: "form-control" %>
  </div>
  <div class="form-group">
    <%= f.submit "投稿", class: "btn btn-outline-secondary btn-block" %>
  </div>
<% end %>